jobs/e2e_node/containerd/containerd-release-1.5/image-config.yaml

```diff
@@ -0,0 +1,9 @@
+images:
+  ubuntu:
+    image: ubuntu-gke-1804-1-17-v20200729 # docker 19.03.2 / containerd 1.2.10
+    project: ubuntu-os-gke-cloud
+    metadata: "user-data</go/src/github.com/containerd/containerd/test/e2e_node/init.yaml,containerd-configure-sh</go/src/github.com/containerd/containerd/cluster/gce/configure.sh,containerd-env</workspace/test-infra/jobs/e2e_node/containerd/containerd-release-1.5/env"
+  cos-stable:
+    image_family: cos-85-lts
```

Comment:
image_family: cos-89-lts

Reply:
Updated. I took the liberty to update the Ubuntu image as well (Docker and Containerd versions are in the comment).